deliver.py

```diff
@@ -144,12 +148,19 @@ def create_delivery_project(base_url, project_name, pi_id, sensitive_data, user,
 
 log.info("Starting delivery of project: {}".format(project))
 
-pi_id = search_pi_id_by_email(base_url=supr_base_url,
-                              email=pi_email,
-                              user=supr_api_user,
-                              key=supr_api_key)
+pi_member_id = map(
+    lambda email: search_pi_id_by_email(
+        base_url=supr_base_url,
+        email=email,
+        user=supr_api_user,
+        key=supr_api_key),
+    [pi_email] + member_email)
+pi_id = pi_member_id[0]
+member_id = pi_member_id[1:]
```

Comment:
Perhaps this variable could be called `member_ids` to make it clearer that it can contain more than one member id?